<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewerPage</class>
 <widget class="QWidget" name="LogViewerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>515</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Log Viewer&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="lvAutoRaiseCheckBox">
     <property name="toolTip">
      <string>Select to show the log-viewer upon new output</string>
     </property>
     <property name="text">
      <string>Show upon new output</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="TextLabel3_2_2">
       <property name="text">
        <string>Error Colour:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stderrTextColourButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select the colour for text sent to stderr</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>316</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Message Filters:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="filtersTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stdout">
      <attribute name="title">
       <string>Standard Output</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="E5StringListEditWidget" name="stdoutFilterEdit" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stderr">
      <attribute name="title">
       <string>Standard Error</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="E5StringListEditWidget" name="stderrFilterEdit" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stdxxx">
      <attribute name="title">
       <string>Both</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="E5StringListEditWidget" name="stdxxxFilterEdit" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5StringListEditWidget</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5StringListEditWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lvAutoRaiseCheckBox</tabstop>
  <tabstop>stderrTextColourButton</tabstop>
  <tabstop>filtersTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
